util/dashboard/fp_007.py

```
@@ -5,29 +5,33 @@
 ## Review your non-RO properties to see if any can be replaced with an RO property using the steps above. Often, a corresponding property will not exist in RO and that is OK.
 ##
 ## ### Implementation
+##
 ## The object and data properties from the ontology are compared to existing RO properties. If any labels match existing RO properties, but do not use the correct RO IRI, this is an error. Any non-RO properties (no label match and do not use an RO IRI) will be listed as INFO messages.
 
 import csv
-import dash_utils
 import os
 import unicodedata
-
 from io import TextIOWrapper
```

Ruff

`io.TextIOWrapper` imported but unused

Remove unused import: `io.TextIOWrapper`

(F401)